jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/projects/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/websocket/ServerEndpointDuplicateVariableURI.java

```diff
@@ -0,0 +1,6 @@
+package src.main.java.io.openliberty.sample.jakarta.websocket;
+
+import jakarta.websocket.server.ServerEndpoint;
+
+@ServerEndpoint("/{varA}/{varB}/{varA}")
```

Comment:
can we add some comments to these test classes indicating where the diagnostic/error is expected and why? Can be brief comments